xgcm/grid.py

```diff
@@ -1080,11 +1080,12 @@ def __init__(
         face_connections : dict
             Grid topology
         coords : dict, optional
-            Explicit specification of axis coordinates, e.g
+            Explicit specification of axis dimensions, e.g
```

Comment:
@rabernat @dcherian I tried to make it clearer here that the input should be dimension names. I guess in retrospect `coords` is a bit of a confusing variable name, but changing that would be a major undertaking. So I figured this would be a quick fix?

Reply:
```suggestion
            Explicit specification dimension positions along axis, e.g
```
?

Reply:
How about `Explicit specification of positions along axis, indicated by a dataset dimension, e.g` to make it even more clear?

Reply:
`Specifies positions of dimension names along axes `X`, `Y`, `Z``?

Reply:
Noice.

Reply:
I think @jdldeauna's input would be valuable here.